src/webots/gui/WbMainWindow.cpp

```diff
@@ -1595,6 +1597,9 @@ void WbMainWindow::upload() {
     } else if (fileName == "cloud_export.jpg") {
       map["foldername"] = WbStandardPaths::webotsTmpPath();
       map["name"] = "thumbnail-file";
+    } else if (fileName.endsWith(".dae", Qt::CaseInsensitive) || fileName.endsWith(".obj", Qt::CaseInsensitive) || fileName.endsWith(".stl", Qt::CaseInsensitive)) {
```

Comment:
Shouldn't we also add the mtl files here?
```suggestion
    } else if (fileName.endsWith(".dae", Qt::CaseInsensitive) || fileName.endsWith(".obj", Qt::CaseInsensitive) || fileName.endsWith(".stl", Qt::CaseInsensitive) || fileName.endsWith(".mtl", Qt::CaseInsensitive)) {
```